vowpalwabbit/cbify.cc

```diff
@@ -386,35 +686,101 @@ void finish_multiline_example(vw& all, cbify&, multi_ex& ec_seq)
 base_learner* cbify_setup(options_i& options, vw& all)
 {
   uint32_t num_actions = 0;
+  uint32_t cb_continuous_num_actions = 0;
   auto data = scoped_calloc_or_throw<cbify>();
   bool use_cs;
+  bool use_reg;  // todo: check
+  bool use_discrete;
 
   option_group_definition new_options("Make Multiclass into Contextual Bandit");
   new_options
       .add(make_option("cbify", num_actions)
                .keep()
                .help("Convert multiclass on <k> classes into a contextual bandit problem"))
-      .add(make_option("cbify_cs", use_cs).help("consume cost-sensitive classification examples instead of multiclass"))
+      .add(make_option("cbify_cs", use_cs).help("Consume cost-sensitive classification examples instead of multiclass"))
+      .add(make_option("cbify_reg", use_reg)
+               .help("Consume regression examples instead of multiclass and cost sensitive"))
+      .add(make_option("cats", cb_continuous_num_actions)
+               .default_value(0)
+               .keep()
+               .help("Continuous action tree with smoothing"))
```

Comment:
is this the best place to define these options? The option group is titled `Make Multiclass into Contextual Bandit` and an option is called 'Consume cost-sensitive classification examples instead of multiclass'?

Reply:
it might make sense to have `continuofy.cc`? Unless a lot of code is being shared across everything that is right now part of cbify

Reply:
Yeah.  I see your point.  There are a number of related regression to cb reductions here.  Not ideal but seems like keeping these together makes sense.